api/src/main/java/javax/mvc/engine/ViewEngineContext.java

```diff
@@ -74,21 +71,6 @@
     Models getModels();
 
     /**
-     * Returns HTTP request object from the Servlet container.
-     *
-     * @return HTTP request object.
-     */
-    HttpServletRequest getRequest();
-
-    /**
-     * Returns HTTP response object from the servlet container. The underlying
-     * output stream should be used to write the result of processing a view.
-     *
-     * @return HTTP response object.
-     */
-    HttpServletResponse getResponse();
-
```

Comment:
I agree with @lefloh. I think we must provide something the engine can write to. This way the view engine can be implemented in a way that works both in servlet and non-servlet environments.

Ozark implements a `MessageBodyWriter<Viewable>` which then looks up the corresponding view engine and calls the `processView()` method. A `MessageBodyWriter` gets an `OutputStream` to write the data to. I guess we should simply expose this stream to the `ViewEngine`? Does this make sense?

Another thing that view engines may have to do is to modify response headers. For setting content type encoding and such stuff for example. The `MessageBodyWriter` is provided with a mutable `MultivaluedMap` for that purpose. We could either also expose this to the view engine via the `ViewEngineContext` interface, or we could add something like `setHeader(String,String)` to it.

Thoughts?